qiita_db/analysis.py

```diff
@@ -266,6 +275,21 @@ def name(self):
         return conn_handler.execute_fetchone(sql, (self._id, ))[0]
 
     @property
+    def portal(self):
```

Comment:
That is not unexpected. Accordingly to the [python documentation](https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references):
> Any identifier of the form __spam (at least two leading underscores, at most one trailing underscore) is textually replaced with _classname__spam


```python
In [21]: class foo(object):
   ....:     def __bar(self):
   ....:         print self
   ....:         

In [22]: x = foo()

In [23]: x.__bar()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-23-883553e2ea7d> in <module>()
----> 1 x.__bar()

AttributeError: 'foo' object has no attribute '__bar'

In [24]: x._foo__bar()
<__main__.foo object at 0x103f9d1d0>
```
which I think is what we want. Adding a check is not ok as it will not allow us to use it to debug the live system.